.gitignore

```diff
@@ -60,7 +60,6 @@ captures/
 .idea/encodings.xml
 .idea/jsLibraryMappings.xml
 .idea/libraries/
-.idea/misc.xml
```

Comment:
Is it needed? Can't we ignore it?

Reply:
It was ignored twice and once is enough ;)